test/disk_usage_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+require 'rack/test'
+
+require 'smart_proxy_pulp_plugin/pulp_plugin'
+require 'smart_proxy_pulp_plugin/disk_usage'
+
+class DiskUsageTest < Test::Unit::TestCase
+  def test_has_path_should_be_true
+    disk_test = ::PulpProxy::DiskUsage.new(:path => '/etc')
+    assert_equal('/etc', disk_test.path)
+  end
+
+  def test_invalid_path_raises_error
+    assert_raises Sys::Filesystem::Error do
+      ::PulpProxy::DiskUsage.new(:path => '/e/t/c')
+    end
```

Comment:
See my comment re: validate_readable above. I realize this is due to the third-party gem, but I prefer to avoid throwing exceptions in constructors -- harder to test, objects left in semi-correct state, etc.